### PR TITLE
Updates template require pattern in viewmodels #1082

### LINKED ISSUE #1082: v75 upgrade - update viewmodel template references
update the template pattern for viewmodels

### PR DESCRIPTION
**Pull request by aj-he**
Update the template require pattern in all viewmodels (except bng related files).

fixes #1082

### CODE REVIEW

File: arches_her/media/js/views/components/plugins/consultation-workflow.js [hunk 1]
```diff
@@ -3,11 +3,12 @@ define([
     'jquery',
     'arches',
     'viewmodels/workflow',
+    'templates/views/components/plugins/consultation-workflow.htm',
     'viewmodels/workflow-step',
     'views/components/workflows/consultation/consultation-dates-step',
     'views/components/workflows/consultation/consultation-map-step',
     'views/components/workflows/consultation/consultations-final-step'
-], function(ko, $, arches, Workflow) {
+], function(ko, $, arches, Workflow, ConsulationWorflowTemplate) {
```
Comment (csmith-he) [inline arches_her/media/js/views/components/plugins/consultation-workflow.js]: Correct spelling here and at line 197

**Review by csmith-he — CHANGES_REQUESTED** [review]
Just a minor spelling change required and then I'm happy to approve

File: arches_her/media/js/views/components/plugins/consultation-workflow.js (hunk 1 above)
Comment (aj-he) [reply arches_her/media/js/views/components/plugins/consultation-workflow.js]: sorted

**Review by csmith-he — APPROVED** [review]
Approved.

**Pull request merged**